### PR TITLE
feat(edit-upload): added the edit upload page

### PR DESCRIPTION
**Pull request by Shruti3004**
## Description

Added the edit upload Page

## How to test

Visit to `/organize/uploads/edit`

## Screenshots

Screenshot from 2021-07-05 13-54-51

### CODE REVIEW

**Comment by Shruti3004** [comment]
@GMishx there is not the api for edit upload but though linked all the other api's linked for it.

File: src/pages/Organize/Uploads/Edit/index.js [hunk 1]
```diff
@@ -14,22 +14,139 @@
...
+
   return (
-    <div className="main-container my-3">
-      <div className="row">
-        <div className="col-lg-8 col-md-12 col-sm-12 col-12">
-          <h1 className="font-size-main-heading">
-            Edit Uploaded File Properties
-          </h1>
-          <br />
+    <>
```
Comment (GMishx) [inline src/pages/Organize/Uploads/Edit/index.js:80]: Is this expected here?

**Comment by GMishx** [comment]
Woops: 
image

**Comment by GMishx** [comment]
> @GMishx there is not the api for edit upload but though linked all the other api's linked for it.

Okay, please open an issue and we can start working on it.

**Comment by Shruti3004** [comment]
> > @GMishx there is not the api for edit upload but though linked all the other api's linked for it.
> 
> Okay, please open an issue and we can start working on it.

Okay sure

**Comment by Shruti3004** [comment]
@GMishx updated

File: src/pages/Organize/Uploads/Edit/index.js (hunk 1 above)
Comment (Shruti3004) [reply src/pages/Organize/Uploads/Edit/index.js:107]: actually this is React fragment I checked the code its fine

**Review by GMishx — APPROVED** [review]
Changes looks good. Working as expected.

**Comment by Shruti3004** [comment]
@GMishx rebased the pr, and fixed the getAllFolders function

**Pull request merged**